windows/src/main/csharp/ch/cyberduck/ui/Views/TransfersWindow.xaml

```diff
@@ -0,0 +1,373 @@
+        </Style>
+    </Window.Resources>
+
+    <DockPanel>
+        <DockPanel Name="ToolBar"
+                   Background="Transparent"
+                   Margin="8,3"
+                   LastChildFill="False"
+                   DockPanel.Dock="Top"
+                   KeyboardNavigation.TabIndex="0"
+                   KeyboardNavigation.TabNavigation="Local">
+            <DockPanel.Resources>
+                <Style TargetType="Button"
+                       BasedOn="{StaticResource {x:Static ToolBar.ButtonStyleKey}}" />
+
+                <Style TargetType="Separator"
+                       BasedOn="{StaticResource {x:Static ToolBar.SeparatorStyleKey}}" />
+            </DockPanel.Resources>
+            <DockPanel.ContextMenu>
+                <ContextMenu>
+                    <MenuItem Header="{xaml:Text Remove,Table=Localizable}"
+                              IsChecked="{Binding ToolbarRemove}"
+                              StaysOpenOnClick="True"
+                              IsCheckable="True" />
+                    <MenuItem Header="{xaml:Text Trash,Table=Transfer}"
+                              IsChecked="{Binding ToolbarTrash}"
+                              StaysOpenOnClick="True"
+                              IsCheckable="True" />
+                    <MenuItem Header="{xaml:Text 'Clean Up',Table=Localizable}"
+                              IsChecked="{Binding ToolbarCleanup}"
+                              StaysOpenOnClick="True"
+                              IsCheckable="True" />
+                    <Separator />
+                    <MenuItem Header="{xaml:Text Resume,Table=Transfer}"
+                              IsChecked="{Binding ToolbarResume}"
+                              StaysOpenOnClick="True"
+                              IsCheckable="True" />
+                    <MenuItem Header="{xaml:Text Stop,Table=Localizable}"
+                              IsChecked="{Binding ToolbarStop}"
+                              StaysOpenOnClick="True"
+                              IsCheckable="True" />
+                    <MenuItem Header="{xaml:Text Reload,Table=Localizable}"
+                              IsChecked="{Binding ToolbarReload}"
+                              StaysOpenOnClick="True"
+                              IsCheckable="True" />
+                    <Separator />
+                    <MenuItem Header="{xaml:Text Open,Table=Localizable}"
+                              IsChecked="{Binding ToolbarOpen}"
+                              StaysOpenOnClick="True"
+                              IsCheckable="True" />
+                    <MenuItem Header="{xaml:Text Show,Table=Localizable}"
+                              IsChecked="{Binding ToolbarShow}"
+                              StaysOpenOnClick="True"
+                              IsCheckable="True" />
+                </ContextMenu>
+            </DockPanel.ContextMenu>
+
+            <StackPanel Orientation="Horizontal"
```

Comment:
This is ugly, but WPF doesn't have an AppBar, or any way of making the toolbar not suck.